### ISSUE TITLE
Built in formatter appears to be interfering with .html files, as well as .twig

### ISSUE THREAD

**Issue by jamesMcMeex** [mblode/vscode-twig-language#13]
Hey, thanks for writing a great extension which I've used a lot over the past 3 months, it's been a great help - much appreciated!

I've noticed an issue with the formatter, likely related to issues already raised - [10](https://github.com/mblode/vscode-twig-language/issues/10) and [6](https://github.com/mblode/vscode-twig-language/issues/6), where when this extension is enabled, the VS Code indentation settings are ignored in any file based on html. When hitting Shift+Alt+F in an html file with the extension enabled, the file is always indented with 4 spaces. 

My VS Code workspace settings are:

```
"editor.tabSize": 2,
"editor.insertSpaces": false,
"editor.detectIndentation": false,
```
These settings should produce the following formatting in a .html file:
<img width="259" alt="screenshot 2018-02-08 14 24 42" src="https://user-images.githubusercontent.com/5237873/36933857-dbf9f022-1ed7-11e8-905c-dac0209745e9.png">

When the extension is enabled, I get the following formatting in a .html:
<img width="251" alt="screenshot 2018-02-08 14 24 27" src="https://user-images.githubusercontent.com/5237873/36933865-f0cbe924-1ed7-11e8-940a-58af8e89075c.png">

If I disable all extensions except this one, I can reproduce this problem.
If I enable all extensions except this one, I do not reproduce the problem.
This issue is also present in .twig files.

I thought it would be worth raising this as it concerns .html files as well .twig files.

**Comment by mblode** [comment]
Hey @jamesMcMeex, I have just pushed an update to Twig Language 2 with a brand new formatter using Prettydiff 3 which brings about some major bug fixes. I have finally fixed this issue with the newest update to Twig Language. Simply set your "twig-language: tabSize" to 0 and it will default to your "editor: tabSIze".  Sorry about the slow development speeds and sub-par communication but hopefully these updates have fixed some of your other problems.
